doc/nrf/app_dev/device_guides/nrf54l/kmu_basics.rst

```diff
@@ -0,0 +1,57 @@
+.. _ug_nrf54l_developing_basics_kmu:
+
+Introducion to KMU key provisioning
+###################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nRF54L devices are equipped with a Key Management Unit (KMU) that facilitates secure and confidential storage of keys.
+This feature is crucial not only for private keys but also for public keys, as the :re:`KMU can directly transfer a key to the CRACEN RAM<ug_nrf54l_crypto_kmu_cracen_peripherals>`.
+Even when keys must pass through addressable RAM, the KMU significantly reduces the risk of key exposure.
+Therefore, you should use KMU for managing secrets whenere possible.
+
+Revocable keys
+**************
+
+Keys can be set as invalid for further use with a revocation policy (RPOLICY), marked as ``revoked``.
+For some technical solutions, a few keys of the same key type (for example, generation 0, 1, 2) might be provisioned.
+At any given time, only one generation should be active.
+If a key generation is compromised, it should be revoked to prevent its use.
+The specifics of the revocation scheme depend on the technical solution, but it is recommended to designate one generation as non-revocable (``locked``) to prevent a Denial of Service (DoS) attack on the key's availability for the solution.
+
+Locked keys
+***********
+
+Locked keys should be permanently available for use, with an RPOLICY of ``locked``.
```

Comment:
Setting key policy as 'locked' disables possibility to invalidate given key.

Reply:
should -> are